tdrs-backend/plg/prometheus/manifest.yml

```diff
@@ -0,0 +1,16 @@
+version: 1
+applications:
+  - name: prometheus
+    memory: 512M
+    disk_quota: 7G
+    instances: 1
+    command: |
+      mkdir /home/vcap/app/prometheus-data &&
+      wget https://github.com/prometheus/prometheus/releases/download/v2.54.1/prometheus-2.54.1.linux-amd64.tar.gz &&
+      tar -zxvf prometheus-2.54.1.linux-amd64.tar.gz &&
+      mv ./prometheus-2.54.1.linux-amd64/prometheus ./prometheus &&
+      mv ./prometheus-2.54.1.linux-amd64/promtool ./promtool &&
+      rm -rf ./prometheus-2.54.1.linux-amd64 && rm -rf prometheus-2.54.1.linux-amd64.tar.gz &&
+      ./prometheus --config.file=/home/vcap/app/prometheus.yml --storage.tsdb.path=/home/vcap/app/prometheus-data --storage.tsdb.retention.time=30d --storage.tsdb.retention.size=6GB  --web.listen-address="0.0.0.0:8080"
```

Comment:
Look into cloud.gov equivalent of ebs storage since app storage is transient

Reply:
[Grafana Mimir](https://github.com/grafana/mimir) supports remote read and write by way of S3. This is probably the only solution we can use in Cloud.gov to support long term metric storage.

Reply:
[Prometheus Remote Storage Integrations](https://prometheus.io/docs/operating/integrations/)